etl/example.rkt

```diff
@@ -0,0 +1,16 @@
+#lang racket
+
+(provide (contract-out
+          [etl (-> hash? hash?)]))          
+
+; TODO Provide contract on the keys and values of the input and output
```

Comment:
I'll try this one, but I can't guarantee enough Racketeerism to get acceptable results :)